package/src/fileSystem.js

```diff
@@ -112,16 +111,14 @@ const save = (folderNamePath, file, fileData, flag) => {
   // proposed start of replacement of for loop
 
   const array = _.range(0, fileDataLength, 1);
-  _.forEach(array, (file, fileName, flag, elementPath, fileData) => {
-    fileName = (file, fileData, flag);
```

Comment:
not sure if it's a good idea to change how it works, because we rely on this method at few places. and our code not ideal - so it can cause some damage...

Reply:
line 116 does not make any meaning,i guess getFileName method is to be used.
@line 115,i dont get the need to have elementPath when we will still redefine it in @line117. flag and file are passed into the save function,so there is no need to be passing it again from the  foreach callback